VatSpace/main.lua

```diff
@@ -66,9 +66,20 @@ local function startGame()
     gv.game_state = 'PLAYING'
 end
 
+local function tryAgain()
+    gv.setVariablesToStartValues()
+    asteroids.create()
+    cats.create()
+    ui.restartUI()
+    startTimer()
+    gv.game_state = 'PLAYING'
+end
+
```

Comment:
Are you sure this function is nesesery? Basically fulfills exactly the same function as the startGame function so I can't see a reason why to not use startGame funtion to restart game.

Reply:
If we use startGame it will create menu again with moving features. Then it looks bad.

Reply:
In my opinion it should always move when you start playing.